src/wazuh_db/helpers/wdb_global_helpers.h

```diff
@@ -295,7 +296,7 @@ int wdb_reset_agents_connection(const char *sync_status, int *sock);
  * @param[in] sock The Wazuh DB socket connection. If NULL, a new connection will be created and closed locally.
  * @return Pointer to the array, on success. NULL on errors.
  */
-int* wdb_get_agents_by_connection_status(const char* connection_status, int *sock);
+int* wdb_get_agents_by_connection_status(const char* connection_status, int *sock, char* node_name);
```

Comment:
Add new parameter to function description.

Reply:
Also, I think you can make `node_name` const.

Reply:
Done!